aws_lambda_powertools/utilities/batch/base.py

```diff
@@ -100,6 +101,26 @@ def process(self) -> List[Tuple]:
         """
         return [self._process_record(record) for record in self.records]
 
+    @abstractmethod
+    async def _async_process_record(self, record: dict):
+        """
+        Async process record with handler.
+        """
+        raise NotImplementedError()
+
+    def async_process(self) -> List[Tuple]:
+        """
+        Async call instance's handler for each record.
+        """
+
+        async def async_process():
+            return list(await asyncio.gather(*[self._async_process_record(record) for record in self.records]))
+
+        # WARNING: Do not use "asyncio.run(async_process())", there are cases in which for some reason the main
+        # loop closes while using, causing the error "Event Loop is closed"
```

Comment:
```suggestion
        # WARNING
        # Do not use "asyncio.run(async_process())" due to Lambda container thaws/freeze, otherwise we might get "Event Loop is closed"
        # Instead, get_event_loop() can also create one if a previous was erroneously closed
        # More: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtime-environment.html#runtimes-lifecycle-shutdown
```

More info for completion for future maintenance.